### PR TITLE
Add A replica of the todolist project

### PR DESCRIPTION
**Pull request by AlyAtalla**
Created a replica of the to do list project with the same code in the to-do-list repo.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      </form>
+      <ul class="nav nav__items" id="list_navv">
+      </ul>
+      <div class="clear">
+        <button type="button" class="btn clear--all" id="clearAllBTN">Clear all completed</button>
+      </div>
+    </div>
+  </main>
+  <script src="index.js"></script>
+</body>
```
Comment (Bhekisile) [inline dist/index.html]: Consider removing line 31. Webpack is taking care of all JavaScript file as a bundler.

**Review by Bhekisile — CHANGES_REQUESTED** [review]
Hi Ali,

Highlights:
Good use of functions and use of ES6.
Good use of modules.

Your code looks clean, you have followed JavaScript best practices. However, there are few suggestions that may improve your code. 

Well done so far!

File: dist/index.html (hunk 1 above)
Comment (AlyAtalla) [reply dist/index.html]: thank you.

**Review by Bhekisile — APPROVED** [review]
Status: Approved ✔️
Hi [@Ali][Ali](https://github.com/AlyAtalla), 👋 👋

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations! 🎉💯🌟

**Pull request merged**